docs/modeling/src/concepts/walletServiceStates.pml

```diff
@@ -1,23 +1,24 @@
-@startuml WalletServiceStates
+@startuml WalletStates
 
-title Wallet Service States
+title Wallet States
 
-[*] --> None
-None --> Future
-None --> Connecting
-Connecting --> Deployed
-Future --> Deploying
-Deploying --> Future
-Deploying --> Deployed
+None --> Future: createFuture
+None --> Connecting: connect
+Connecting --> Deployed: waitForSuccess
+Future --> Deploying: waitForDeploy
```

Comment:
Future --> None: disconnect

Reply:
We can disconnect from each state... Adding this can make this image a little unreadable.